Tests/FoundationTests/DateExtensionsTests.swift

```diff
@@ -860,4 +860,58 @@ final class DateExtensionsTests: XCTestCase {
 		XCTAssertEqual(date, dateFromUnixTimestamp)
 	}
 	
+    func testRandom() {
```

Comment:
It would be great if we could just combine all the tests below into this single test.
I would like to maintain a 1 : 1 ratio between tests and extensions. This way when they fail we are clear of the cause.

Reply:
OK, thanks. I'll do it